packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,199 @@
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+  <FormElement.Instructions>
+    <span>
+      Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example text for
+      extra panel for questionnaires. Example text for extra panel for questionnaires
+    </span>
+  </FormElement.Instructions>
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <Input id={idForLabel} placeholder="Form placeholder" aria-describedby={ariaDescribedBy} />
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>This is description text</span>
+  </FormElement.Description>
+  <FormElement.Error>There was an error</FormElement.Error>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <AccessibilityExample />
+</FormElementStory>
+
+### Basic Input Components
+
+The `<FormElement>` can be used with different components to fulfill a variety of input needs. This can be achieved by including basic input components such as the `<Select>`, `<Textarea>`, `<Checkbox>`, etc. as `children` for the `<FormElement.Content>`
+
+#### Textarea
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    <Textarea defaultValue="Hello world">
+  </FormElement.Content>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <FormElement hasFieldSet label="Form Label">
+    <FormElement.Help>
+      Give me some help. <a href="wegalvanize.com">Learn more</a>.
+    </FormElement.Help>
+    <FormElement.Content>
+      <Textarea defaultValue="Hello world" />
+    </FormElement.Content>
+  </FormElement>
+</FormElementStory>
+
+#### Checkbox
+
+```jsx
+const optionsArray = ["Black Panther", "Wonder Woman", "Spiderman", "The Incredibles", "Thor", <span>test</span>];
+
+const getCheckboxOptions = ariaDescribedBy =>
+  optionsArray.map(hero => (
+    <Checkbox
+      ariaDescribedBy={ariaDescribedBy}
+      key={hero}
+      onChange={() => {}}
+      isDisabled={false}
+      size={Checkbox.types.size.MEDIUM}
+    >
+      {hero}
+    </Checkbox>
+  ));
+
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getCheckboxOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+</FormElement>;
+```
+
+##### Result
+
+<FormElementStory>
+  <CheckboxExample />
+</FormElementStory>
+
+### Complex Input Components
+
+The `<FormElement />` can handle a variety of components to further increase the users indivudal needs. This can range from a `<DatePicker />` component to a `<ListBox />` component. The following code snippet is an example of the `ComponentLabel` this example allows users to use the `label` property of the `<FormElement/ />`. Users can set the label property to equal an entire component, in this particular example making use of the `<Heading />` component.
+
+#### Component Label
```

Comment:
Actually, now that you've got these examples split out, I think it makes more sense to do the Component Label example as its own `###` level heading.  It's not really a "complex component" example, just another more advanced option.